### ISSUE TITLE
Update bbUI references

### ISSUE THREAD

**Issue by tneil** [blackberry/bbUI.js#412]
Once the v0.9.5 branch is ready to merge into master the wiki references for where we show how to link in the files need to be updated to use the minified versions of the file names.

wiki pages to update:
- [Toolkit Initialization](https://github.com/blackberry/bbUI.js/wiki/Toolkit-Initialization)
- [Application Structure](https://github.com/blackberry/bbUI.js/wiki/Application-Structure)
